src/listTags.js

```diff
@@ -77,8 +77,8 @@ function createTagIndex(noteFolderPath) {
           let tagIndex = {};
           for (let i = 0; i < files.length; i++) {
             if (files[i] != null && files[i]) {
-              const parsedFrontMatter = matter(files[i].contents);
```

Comment:
Do we still need to get the frontmatter from the contents key?

Reply:
Yes. This line was moved into parseFrontMatter function. (L.102)
https://github.com/patleeman/VSNotes/pull/62/files#diff-1329968ba9ffacb8c53f15d8692ef587R102

The reason for passing the file object to the parseFrontMatter function is to get front matter from `contents` and to display the `path` as error messages.

Reply:
ah i totally missed that. Thanks for clarifying.